application/helpers/export_helper.php

```diff
@@ -3144,6 +3144,11 @@ function writeXmlFromArray(XMLWriter $xml, $aData, $sParentKey = '')
 {
     $bCloseElement = false;
     foreach ($aData as $key => $value) {
+        if ($key == "@attributes") {
```

Comment:
Is this the right place for this filter?
This function is a writer. Should also have the logic for filtering the sources?

Maybe here better fit?
https://github.com/LimeSurvey/LimeSurvey/pull/3845/files#diff-1aecd900cde469d4c2ba64ab989062a075ed3ff9378180051d812b127a929dfeR3205

Reply:
> Is this the right place for this filter? 

I don't know :)

You're right about the place it was added